api/routes/openAPI/helpers.js

```diff
@@ -36,16 +36,29 @@ module.exports.schema = {
   })
 };
 
-module.exports.requiresAuth = openAPI => {
+module.exports.requiresAuth = (
+  openAPI,
+  { has401 = true, has403 = true } = {}
+) => {
```

Comment:
Make it so OpenAPI components can opt out of 401 or 403 statuses instead of being required to have both.